services/user.py

```diff
@@ -0,0 +1,50 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.models import User
+
+
+def create_user(username: str,
+                password: str,
+                email: str = None,
+                first_name: str = None,
+                last_name: str = None) -> None:
+    get_user_model().objects.create_user(
+        username=username,
+        password=password
+    )
+    user = get_user_model().objects.get(username=username)

```

Comment:
`create_user` returns created object. You don't need to make get query separately